server/service/integration_core_test.go

```diff
@@ -6486,6 +6486,58 @@ func (s *integrationTestSuite) TestTeamPoliciesTeamNotExists() {
 	s.DoJSON("POST", fmt.Sprintf("/api/latest/fleet/teams/%d/policies/delete", 9999999), deleteTeamPoliciesRequest{IDs: []uint{1, 1000}}, http.StatusNotFound, &deleteTeamPoliciesResponse)
 }
 
+func (s *integrationTestSuite) TestTeamPolicyPendingScript() {
```

Comment:
Let's add a test for global policies too. And this one should be moved to integration_enterprise_test.go (because it's for teams which is a premium/enterprise feature).

Reply:
Good to know! I'll probably end up deleting this given I move the logic into the datastore, I can test the same scenario in the datastore tests.